packages/component-library/components/Button/styles.scss

```diff
@@ -51,19 +52,18 @@ $caButton-verticalPaddingForm: calc(
   // equivalent to :enabled on form elements, but also allows these styles to be
   // applied to elements that have no enabled/disabled states.
   &:not(:disabled) {
-    background: $white;
-    border-color: add-tint($ca-palette-ink, 80%);
-    color: $ca-palette-ink;
+    background: $kz-color-white;
+    border-color: $kz-color-wisteria-500;
+    color: $kz-color-wisteria-800;
 
-    &:hover,
-    &:active {
-      background: $ca-palette-stone;
-      border-color: add-tint($ca-palette-ink, 60%);
+    &:hover {
+      background: $kz-color-wisteria-100;
+      border-color: $kz-color-wisteria-700;
```

Comment:
Beautiful attention to detail 😍 I notice that the border colours and their states (hover/active/etc) align to the new Zen UI kit.